__generator__/builtin.yml

```diff
@@ -1338,4 +1338,25 @@ digest.hash_xxh64:
   on: [RECV, HASH, HIT, MISS, PASS, FETCH, ERROR, DELIVER, LOG]
   arguments:
     - [STRING]
+  return: STRING
+
+digest.hash_xxh32_from_base64:
+  reference: "" # TODO: fill URL after documentation found
```

Comment:
Documentation is here maybe: https://www.fastly.com/documentation/reference/vcl/functions/cryptographic/digest-hash-xxh32-from-base64/